.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.0
```

Comment:
why doing it like that instead of setting `TargetRubyVersion: 2.5` in the `.rubocop.yml` file?

Reply:
This file is picked up by Rubocop automatically. See the documentation here: https://github.com/bbatsov/rubocop/blob/52d03b4b1d9296441360a4e4bbe0498be1be1141/manual/configuration.md#setting-the-target-ruby-version

Reply:
I know, my question was why doing it this way and not setting the target in the config file... I am actually not sure where it is less likely that we forget to change it in the next update :thinking: